python/src/file.rs

```diff
@@ -113,6 +115,51 @@ impl LanceColumnMetadata {
     }
 }
 
+#[pyclass(get_all)]
+#[derive(Clone, Debug, Serialize)]
+pub struct LanceFileStatistics {
```

Comment:
I think the `Lance` prefix is probably unnecessary.
```suggestion
pub struct FileStatistics {
```

Reply:
actually, I think this naming is a convention in the current code base:
```
    LanceBufferDescriptor,
    LanceColumnMetadata,
    LanceFileMetadata,
    LanceFileStatistics,
    LancePageMetadata,
```